api/code-lists.go

```diff
@@ -19,6 +19,14 @@ func (c *CodeListAPI) getCodeLists(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	for _, item := range codeLists.Items {
+		if err := item.UpdateLinks(c.apiURL); err != nil {
```

Comment:
just a thought and ignore if it doesn't apply, but if this fails will it let us know what it failed on? i.e which missing/incorrect code.